### ISSUE TITLE
images/ and sounds/ should have the same parent folder

### ISSUE THREAD

**Issue by kevinbarabash** [Khan/live-editor#441]

**Comment by Gigabyte-Giant** [comment]
Are you suggesting the `images/` and `sounds/` directory be children of a `resources/` directory (the name could be anything, obviously)?

**Comment by kevinbarabash** [comment]
That should work.  We also have two copies of all of the images.  I need to have a look at webapp to see where we're serving those files from, but it would be nice if could only have one copy of those in the repo.

**Comment by Gigabyte-Giant** [comment]
When students use images in their projects on Khan Academy, aren't they loaded from the appropriate child-directories of `https://www.kasandbox.org/third_party/javascript-khansrc/live-editor/build/images/`?

**Comment by kevinbarabash** [comment]
Yup, that's correct.  I need to track down how those images get uploaded to `kasandbox.org` so that if we change their location they will still get uploaded to the correct spot.  The reason why this is necessary is that their location on `kasandbox.org` is hardcoded in webapp.

**Comment by Gigabyte-Giant** [comment]
That makes sense...

**Comment by pamelafox** [comment]
(Assuming things haven't changed) they're served off an S3 folder, and our varnish proxy makes them appear to be served off of kasandbox.org.

**Comment by DozenalFTW** [comment]
Why can't we use program thumbnails in PJS programs like we can in HTML?
